docs/intro/glossary.md

```diff
@@ -0,0 +1,241 @@
+# Glossary
+
+This glossary provides definitions for common terminology used in the SDK,
+focusing on microgrid components, metrics, measurements, and power-related
+terms.
+
+## Common Acronyms
+
+### AC
+
+Alternating current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
```

Comment:
```suggestion
Alternating Current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
```

Reply:
Why the caps here (and in the other acronym definitions)?

Reply:
Because I wanted to somehow highlight what the acronym stands for. I initially made an optional global comment to check if we want to go that way, then I made suggestions where we can possibly updated if the global suggestion was taken. So feel free to discard this and similar suggestions if they are not suitable